people/Hangmster/page.en.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Hangmster
+info:
+    alias: 仓鼠
+    location: Shijiazhuang, Hebei
+---
+
+Hangmster is an ally of us.
+
+They often interacted with friends at Twitter/X, and they were a barrel of laughs. They would also give warm hugs when we needed, console us when we were in anxiety, and mourn for our departed friends.
+
+Although they seldom mentioned what they loved. However, according to their tweets, they would be attracted by cute things such as little cats. They are fond of Hatsune Miku as well. In the Hatsune Miku 15th anniversary, they expressed their happiness by drawing an illustration:
```

Comment:
They seldomly mentioned what they loved.
But according to their tweets, they were interested in cute things such as cats.
They were fond of Hatsune Miku as well.
On the Hatsune Miku 15th anniversary, they expressed their happiness by drawing an illustration: